### ISSUE TITLE
This is not worth a slam try

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#608]
0C2E9BACFF3C44572DF4AA1DB224
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/31e5bc7e-a00d-42a4-b4ec-a4ee1121d9db)

North is a passed hand so 4D doesn't really get us anywhere

Now change the bidding a bit, and we get another problem

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/4c23ca93-8c2c-476c-884a-9fe237bd1f8e)

I can't construct a hand where North can make a penalty double of 4D
so I think GIB is right in defining this as a responsive double
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/26382357-b433-4a05-935f-2679702b2013)

**Comment by ThorvaldAagaard** [comment]
This is not worth a slam try
0FE2656A00E575233AAA7303A7C5
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/d9fb310b-9cb1-4e0f-b23f-4379871a085f)

Or in other words, how would a hand in North, that will bid anything else than 4H look like?
And playing with GIB (BBO-BEN) this can happen
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/abedbc84-32dd-46a5-8026-a33bfc760753)
Only 5 down

**Comment by ThorvaldAagaard** [comment]
Another example
00B3DF0B9449802412BFC6F2DA90
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/14faff8d-a207-4f07-83f2-5f70cf2a741f)

**Comment by ThePokerDude** [comment]
Why is BBA not following it's description?
3NT: maximum 4s 10 points
3s: 5+ spades 9+ points
![image](https://github.com/user-attachments/assets/92d12595-098d-45c9-bf3a-b36d771a8447)
![image](https://github.com/user-attachments/assets/d78859bc-5f5e-472d-8b07-69b6f85b7667)

**Comment by EdwardPiwowar** [comment]
After a high 3d opening it is hard to expect bids that describe hands precisely. In this case BBA chooses 3NT as having the best chance of being realized. Multiple hand generation and DD analysis could give a better result but at the cost of speed.

**Comment by ThorvaldAagaard** [comment]
Souths 3H is wrong. It is a perfect T/O D, and North has an easy 4S

**Comment by EdwardPiwowar** [comment]
Maybe yes but if more probable hands
![image](https://github.com/user-attachments/assets/a856fefb-0a7d-454e-891f-a2a98fd0ab99)

**Comment by ThorvaldAagaard** [comment]
Over 4D North still has to act. X is a possibility, and here 3N will not make.
Over preemps you should not bid a 5-card suit if you can play in all 3 suits,

**Comment by ThePokerDude** [comment]
> After a high 3d opening it is hard to expect bids that describe hands precisely. In this case BBA chooses 3NT as having the best chance of being realized. Multiple hand generation and DD analysis could give a better result but at the cost of speed.

Obviously I don't understand how description and bidding are connected.
I would expect that the description shows the internal checks - so if a description states 4 spades exactly the engine checks the hand if it has 4 spades exactly. 
We check all the possible bids for a match between description and hand.
Only if we don't find a match we choose the "most matching bid".

In this case 3s shows 9+hcp, 5+spades. So the hand fulfills all conditions of the description.
3nt does not fulfill all conditions.

That's why I'd expact BBA to choose 3s.
Obviously it works differently.

**Comment by EdwardPiwowar** [comment]
Yes, BBA tests for fit to predefined bids. May make some exceptions. Explanation is the initial bid definition.
GIB
![image](https://github.com/user-attachments/assets/105d4aa8-87cb-45d1-9a70-4361150120ff)
WB5
![image](https://github.com/user-attachments/assets/f6458c7c-15e5-476f-b7b2-845fb05cf7d0)

It seems that double is the most appropriate bid for S
![image](https://github.com/user-attachments/assets/971c11d2-07a8-437f-bbf2-e94afdb10d96)
